Makefile.am

```diff
@@ -740,10 +741,13 @@ EXTRA_DIST += \
 	man/glib-event-glue.c
 
 # ------------------------------------------------------------------------------
+rootlibexec_LTLIBRARIES += \
+	libshared.la
```

Comment:
i think we should give this a more specific name, i.e. "libsystemd-shared" or so...